frontend/src/app/workflows/workflows.component.ts

```diff
@@ -1,15 +1,212 @@
-import { Component, OnInit } from '@angular/core';
+import {Component, NgZone, OnInit, ViewChild} from '@angular/core';
+
+import Ajv from 'ajv';
+import {ToastService} from '../toast.service';
+import {WorkflowsService} from './workflows.service';
+import {GenerateResult} from './model/GenerateResult';
+import {YamlHelper} from './model/helper/yaml.helper';
+import {IOutputData, SplitComponent} from 'angular-split';
+import {createMapFromAnswer} from './model/helper/map.helper';
+import {workflowsSchema} from './model/helper/workflows.schema';
+import {msExample, newWorkflowExample, pagesExample, pmExample} from './model/examples';
 
 @Component({
   selector: 'app-workflows',
   templateUrl: './workflows.component.html',
   styleUrls: ['./workflows.component.scss']
 })
 export class WorkflowsComponent implements OnInit {
+  @ViewChild('split') split!: SplitComponent;
+
+  public content!: any;
+  public codemirrorOptions: any;
+
+  public generateResult!: GenerateResult;
+
+  public currentExampleDesc: string = 'Select example';
+  public examplesList = ['Empty workflow', 'Data Modelling', 'Microservices', 'Pages'];
+
+  public showIframeHider = false;
+  public newPageIndex!: number;
+  public currentDisplay: 'pages' | 'objects' | 'class' = 'pages';
+
+  private ajv!: Ajv;
+  private validate!: any;
+  private yamlHelper!: YamlHelper;
+  private loading: boolean = false;
 
-  constructor() { }
+  constructor(private toastService: ToastService,
+              private zone: NgZone,
+              private fulibWorkflowsService: WorkflowsService) {
```

Comment:
Please put every service on a new line (also in other files):

```suggestion
  constructor(
    private toastService: ToastService,
    private zone: NgZone,
    private fulibWorkflowsService: WorkflowsService,
  ) {
```

Reply:
Done sire